rpcs3/Emu/RSX/Program/ProgramStateCache.cpp

```diff
@@ -404,11 +405,10 @@ usz fragment_program_utils::get_fragment_program_ucode_size(const void* ptr)
 	{
 		const v128 inst = v128::loadu(instBuffer, instIndex);
 		bool isSRC0Constant = is_constant(inst._u32[1]);
-		bool isSRC1Constant = is_constant(inst._u32[2]);
-		bool isSRC2Constant = is_constant(inst._u32[3]);
+		bool isSRCPairConstant = is_pair_constant(inst._u64[1]);
```

Comment:
This doesn't make much sense (architecturally) and is confusing to read. Just replace is_constant with is_any_src_constant function that evaluates all 3. What we're checking is that any of the instruction's inputs are literal constants and not registers, which usually means to skip the next 16 bytes.